src/api/wayfire/signal-definitions.hpp

```diff
@@ -595,6 +595,23 @@ struct view_geometry_changed_signal : public _view_signal
     wf::geometry_t old_geometry;
 };
 
+/**
+ * name: view-change-viewport
```

Comment:
viewport and workspace used to be used interchangeably, however, I think for new code it should only be 'workspace'. Since we're changing things I think now is a good time to rename this signal.

Reply:
Done.